docs/sphinx/source/conf.py

```diff
@@ -158,6 +158,7 @@
     ],
     "use_edit_page_button": True,
     "show_toc_level": 1,
+    "footer_items": ["copyright", "sphinx-version", "sidebar-ethical-ads"],
```

Comment:
default: `"footer_items": ["copyright", "sphinx-version"]`

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/sections.html#the-footer